api/app/controllers/mno_enterprise/jpi/v1/app_reviews_controller.rb

```diff
@@ -46,49 +42,64 @@ def update
     def destroy
       @app_review.destroy
       after_save
-
+      @app_review = nil
       render :show
     end
 
     private
 
-    # scope the app_reviews for the children controller
-    # may be overriden
-    def scope_app_reviews
-      @app_reviews.where(reviewable_id: current_app.id)
-    end
-
     # perform some additional actions if new review was created
     # may be overriden
     def after_save
       @average_rating = current_app.reload.average_rating
     end
 
     def current_app
-      @app ||= MnoEnterprise::App.find(params[:id])
+      @app ||= MnoEnterprise::App.find_one(params[:id])
+    end
+
+    def orga_relation
+      @orga_relation ||= MnoEnterprise::OrgaRelation.where(organization_id: organization_id, user_id: current_user.id).first
     end
 
     def ensure_app_exists
       render_not_found('App') unless current_app.present?
     end
 
+    def ensure_orga_relation_exists
+      render_not_found('OrgaRelation', "organization_id: #{organization_id}, user_id: #{current_user.id}") unless orga_relation.present?
+    end
+
     def find_review
-      @app_review = review_klass.find(params[:review_id])
+      @app_review = review_klass.find_one(params[:review_id])
       unless @app_review.user_id == current_user.id
         return render json:{ errors: {message: "Review not found (id=#{params[:review_id]})", code: 404} }, status: :not_found
       end
     end
 
     def review_klass
-      MnoEnterprise::AppReview
+      Review
+    end
+
+    def initial_scope
+      review_klass
+    end
+
+    def root_params
+      symbol = self.class.name.demodulize.chomp('Controller').underscore.singularize.to_sym
```

Comment:
[Yes you can!](http://www.rubydoc.info/gems/json_api_client/JsonApiClient/Resource#table_name-class_method)

[We've been  using it already](https://github.com/maestrano/mno-enterprise/blob/4.0/core/app/models/mno_enterprise/tenant.rb#L27-L29)